dashboard/src/components/Button.tsx

```diff
@@ -0,0 +1,57 @@
+import React from "react";
+import styled from "styled-components";
+
+interface Props {
+  disabled?: boolean;
+  children: React.ReactNode;
+  onClick: () => void;
+}
+
+const Button = ({ children, disabled, onClick }: Props) => {
```

Comment:
Hey! Just for a bit of quick advice on typing React components. There's no need to manually type the children prop, as React.FC can do it for you with best practices and support for any future change on that prop 👍 

```suggestion
const Button: React.FC<Props> = ({ children, disabled, onClick }) => {
```

On the other hand if you need to type a component that will not use the children prop you can use React.FunctionComponent that will prevent from accidentally passing children to that component

Reply:
Thanks! Didn't know that this was a thing, will refactor it.